README.md

```diff
@@ -19,29 +19,53 @@ developers of new software pick some non-interfering values.
 
 Bits are numbered from 0 to 31, least-significant bit (LSB) to
 most-significant (MSB). For example, if only mark bit number 3 is set,
-the overall packet mark is 0x8. For search engine discoverability, the
-full mark value with individual bits set is also listed in the form
-that people are likely to search for.
+the overall packet mark is 0x8.
 
-| Bit | Mark value | Software |
+| Bit(s) | Mark value | Software |
```

Comment:
nit: Would `Mark mask` be more appropriate given that multiple bits covered by this value are handled by the corresponding software?